From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Max Filippov <jcmvbkbc@gmail.com>
Date: Fri, 24 Nov 2023 23:22:31 -0800
Subject: [PATCH 37/59] gpio: gpio-mmio: add support for esp,esp32-clk-gpio

gpio-mmio can be used to drive gpio-style clock gates on esp32xx SoCs.
Add esp32-clk-gpio OF id for that.

Signed-off-by: Max Filippov <jcmvbkbc@gmail.com>
(cherry picked from commit d5bee79b342351075e378382a03b2f2ab3a277f3)
---
 drivers/gpio/gpio-mmio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/gpio-mmio.c b/drivers/gpio/gpio-mmio.c
index e9c531eef4521..984dd2ed26176 100644
--- a/drivers/gpio/gpio-mmio.c
+++ b/drivers/gpio/gpio-mmio.c
@@ -722,6 +722,7 @@ static const struct of_device_id gpio_mmio_of_match[] = {
 	{ .compatible = "ni,169445-nand-gpio" },
 	{ .compatible = "intel,ixp4xx-expansion-bus-mmio-gpio" },
 	{ .compatible = "opencores,gpio" },
+	{ .compatible = "esp,esp32-clk-gpio" },
 	{ }
 };
 MODULE_DEVICE_TABLE(of, gpio_mmio_of_match);
